server/.env.example

```
@@ -12,3 +12,8 @@ AI_ALLOW_INSECURE_PRIVATE_URL=false
 MEMOBIRD_ACCESS_KEY=
 MEMOBIRD_BASE_URL=
 MEMOBIRD_TIMEOUT=30s
+PLUGIN_ROOT=.plugins
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PLUGIN_ROOT key should go before the PORT key

(UnorderedKey)